rio_cogeo/cogeo.py

```diff
@@ -159,7 +159,12 @@ def cog_translate(
                     dst_kwargs["blockxsize"] = tilesize
                     dst_kwargs["blockysize"] = tilesize
 
-            vrt_params = dict(add_alpha=True, dtype=dtype)
+            vrt_params = {
+                "add_alpha": True,
+                "dtype": dtype,
+                "width": src_dst.width,
+                "height": src_dst.height,
```

Comment:
Will the width and height always be unchanged?

Reply:
no, when we do `--web-optimized` we will update those with `warped` ones

Reply:
Yeah so these are the defaults to be updated later if needed, sounds good